addons/binding/org.openhab.binding.miio/README.md

```diff
@@ -909,6 +926,15 @@ e.g. `smarthome:send actionCommand 'upd_timer["1498595904821", "on"]'` would ena
 | led              | Switch  | Wifi led                            |
 
 
+### Mi Smart Plug (<a name="chuangmi-plug-hmi205">chuangmi.plug.hmi205</a>) Channels
+
+
+| Channel          | Type    | Description                         |
+|------------------|---------|-------------------------------------|
+| power            | Switch  | Power                               |
+| temperature      | Number  | Temperature                         |
```

Comment:
Any plans for using UoM with this binding? :slightly_smiling_face:

Reply:
hmmm, not really high on my list of wishes to be honest.

Reply:
thanks